### DISCUSSION TITLE
Issues Dumping Galaxian (Japan) for NES

### DISCUSSION THREAD

**Discussion by LNRC** [sanni/cartreader#691] [Need help]
Hello everyone!

I'm having difficulty dumping the game Galaxian (Japan). I think I just need a few alterations to my mapper settings, but I don't know how to make these changes on my own.

The cart reader correctly identifies my game as Galaxian. However, the given mapper settings don't seem to match what I find in the various online databases linked on the wiki.

This is what I get from the Sanni:
Galaxian (Japan).nes
MAPPER: 0
PRG SIZE: 16K
CHR SIZE: 8K
RAM SIZE: 0K

When I try to read the iNES rom, this is what I receive
CRC32... E5D11921 -> Not found

Looking at NesCartDB, the PRG and CHR sizes are listed differently
https://nescartdb.com/profile/view/1808/galaxian
PRG SIZE: 8K
CHR SIZE: 8K

Currently, I don't see an option to dump any PRG size smaller than 8K, at least for Mapper 0.

Looking at the NES header database, I get even more different information
http://nes.dnsabr.com/entity/view?id=811
PRG SIZE: 16K
CHR SIZE: 0K

I tried changing the CHR size to 0K since that was an option, and these were the results:
CRC32... 6BFE4FF1 -> Not found

Comparing the original dump (PRG 16K/CHR 8K) and the altered CHR dump (PRG 16K/CHR 0K), both files are 1.5x the size of the original rom. The header appears to repeat partway through. However, I'm not sure what portion of the data I need to cut out for my dump to match the original rom.

Please let me know if there's any further information I can provide. I hope someone can help!

NOTE: I have already cleaned this cartridge many times, it works fine on a physical NES, and my cart reader works fine on other cartridges

**Comment by sanni** [comment]
In the Cart Reader code the smallest size for PRG is 16K so it can't handle games with 8K PRG like Galaxian.
So it dumps: iNES-PRG-PRG-CHR
You need to delete the block from 0x10 to 0x200F.
